lib/util/tXml.js

```diff
@@ -876,6 +903,7 @@ shaka.util.TXml.knownNameSpaces_ = new Map([]);
  * @typedef {{
  *   name: string,
  *   id: ?string,
+ *   t: ?number,
```

Comment:
It looks like this field is not needed in type definition, as you're using it internally to find out proper index of element, right?

Reply:
That is correct but we need to have it like the id or the other to get information in the code here https://github.com/shaka-project/shaka-player/blob/cd06997ab377f1fc1882206bb7dd5824c21d2325/lib/util/tXml.js#L786

Reply:
@tykus160  I'm not seeing other way to do it

Reply:
same as position or id it's part of the path documentation so should be part of the model

Reply:
I see now, let's keep it like that for now.